Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -170,8 +170,13 @@ class TopTabsViewController: UIViewController {
     
     func privateTabTapped() {
```

Comment:
Nit: I just noticed that this method is actually fired when private mode is toggled, not when a private tab is selected. Probably worth renaming to reflect that. Same with the `tabsTapped` method above.